content/guides/references/configuration.md

```diff
@@ -535,8 +536,8 @@ send a `503` status code. As a convenience it also sets a
 
 ### devServer / devServerConfig
 
-The `devServer` function is a required [`component`](#component) runner specific
-option, and allows you to register a component testing dev server. It receives a
+The `devServer` function is a required [`component`](#component) testing option,
```

Comment:
Anywhere it said "runner specific option" can you say "testing specific option"? I feel the "specific" is important.